rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init

```diff
@@ -174,6 +174,7 @@ BACKUP_SUFFIX="`date +%Y%m%d%H%M%S`.ldif"
 # Other
 TIMEOUT="60"    # Max time to stop process
 FD_LIMIT="2048" # Max file descriptor
+VM_LIMIT="108490720" # Max virtual memory
```

Comment:
We could make the ulimit change a part of the script that configures the mdb size, but it not called from every action in the init script (and that would not make sense to add it).

We can:

* Put a `ulimit -v` before every slap* call, it could be
  * a hardcoded value
  * computed from a script that uses the current mdbsize, which would be called in migration scripts and in every action fo the init script
* The only other option I see is to configure virtual memory limit for the root user in limits.conf. I don't think we want to to it silently, or even do it at all.